psycop/common/feature_generation/sequences/patient_loaders.py

```diff
@@ -103,6 +103,7 @@ def load_date_of_birth_df() -> pl.DataFrame:
     def get_split(
         event_loaders: Sequence[EventDfLoader],
         split: SplitName,
+        patient_column_names: dict[str, str] | None = None,
```

Comment:
```suggestion
        patient_column_names: PatientColumnNames | None = None,
```

This indicates to the user that they should use the PatientColumnNames object, rather than a dict :-)